subtrack.MAUI/Pages/SubscriptionDetails.razor

```diff
@@ -138,4 +144,18 @@ else
         await subscriptionService.Delete(Id);
         navigationManager.NavigateTo("/", replace: true);
     }
+    private string GetOccurance()
```

Comment:
"occurrence"

can add the suffix "Text" to be explicit that it returns the occurrence in text format